ssg/_build_ovals.py

```diff
@@ -242,7 +242,18 @@ def _check_is_loaded(loaded_dict, filename, version):
 
 
 def _check_oval_version_from_oval(xml_content, oval_version):
-    oval_file_tree = ElementTree.fromstring(oval_header + xml_content + footer)
+    try:
+        argument = oval_header + xml_content + oval_footer
+        oval_file_tree = ElementTree.fromstring(argument)
+    except ElementTree.ParseError, p:
+        line, column = p.position
+        lines = argument.splitlines()
+        before = '\n'.join(lines[:line])
+        column_pointer = ' ' * (column - 1) + '^'
+        sys.stderr.write(
```

Comment:
I would personally use print("blabla", file=sys.stderr) because it will do the final newline for you but it's not a big deal.

Reply:
Yeah, it makes sense, it's just that `sys.stderr.write` is used across the file. I am fine with changing it everywhere if I change it here :) Should I?